pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
This needs to be setuptools if we are indeed going back to the original approach of pushing to pypi.

Reply:
Sure, you prefer setuptools? The build backends (setuptools, hatch, flit, pdm,...) should work interchangeably.
I chose hatch because pypa recommends it in the packaging tutorial.
Actually you can also do hatch build and hatch publish.

Reply:
Yep, it's also a manner of convenience since I'm familiar with it and will be the one debugging things when pushing to pypi.